proto/tendermint/types/types.proto

```diff
@@ -57,28 +57,36 @@ message BlockID {
 // Header defines the structure of a Tendermint block header.
 message Header {
   // basic block info
-  tendermint.version.Consensus version  = 1 [(gogoproto.nullable) = false];
-  string                       chain_id = 2 [(gogoproto.customname) = "ChainID"];
-  int64                        height   = 3;
-  google.protobuf.Timestamp    time     = 4 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+  tendermint.version.Consensus version                  = 1 [(gogoproto.nullable) = false];
+  string                       chain_id                 = 2 [(gogoproto.customname) = "ChainID"];
+  int64                        height                   = 3;
+  uint32                       core_chain_locked_height = 4;
```

Comment:
Let's do not change fields order so Tendermint headers will be still compatible which could be important for IBC. We could add our fields with some gap in indices, like since 10000

Reply:
Yes, please! Proto fields should be appended only or removed entirely.